src/rank_kernel.patch

```diff
@@ -164,15 +164,28 @@ index da785eaf2d..9125990519 100644
  };
  
  class RankQuantileMetaFunction : public RankMetaFunctionBase<RankQuantileMetaFunction> {
-@@ -380,7 +380,7 @@ class RankQuantileMetaFunction : public RankMetaFunctionBase<RankQuantileMetaFun
-   }
+@@ -374,7 +375,7 @@ class RankQuantileMetaFunction : public RankMetaFunctionBase<RankQuantileMetaFun
+   static RankerType GetRanker(const RankQuantileOptions& options) { return RankerType(); }
```

Comment:
what does this do?

Reply:
(I wonder if this is somehow needed by vendoring or orthogonal)

Reply:
This is a diff of a diff (it's the patch's diff)
This is just because I've regenerated the patch with the latest upstream changes not a change on what we apply

Reply:
Ah ok - surprising I applied this without updating the base arrow tar ball and it still works